util/options_helper.cc

```diff
@@ -1201,10 +1209,80 @@ Status GetPlainTableOptionsFromString(
   if (!s.ok()) {
     return s;
   }
-  return GetPlainTableOptionsFromMap(table_options, opts_map, 
+  return GetPlainTableOptionsFromMap(table_options, opts_map,
                                      new_table_options);
 }
 
+Status GetMemTableRepFactoryFromString(
+    const std::string& opts_str,
+    MemTableRepFactory** new_mem_factory) {
+  std::vector<std::string> opts_list = StringSplit(opts_str, ':');
+  size_t len = opts_list.size();
```

Comment:
I think for now we can return Status::InvalidArgument if len <= 0 || len > 2, instead of handling this case in in every condition